packages/UI/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,527 @@
+          constitutes acceptance of these terms.
+        </p>
+      </div>
+    );
+    const ToSFooter = (
+      <div className="flex h-10 w-full items-center justify-between font-nunito-sans">
+        <div className="flex items-center justify-center">
+          <Checkbox />
+          <p className="ml-2">I've read the terms of services</p>
+        </div>
+        <Close>
+          <Button buttonType="primary" buttonSize="standard">
+            Submit
+          </Button>
+        </Close>
+      </div>
+    );
+    // Role selection modal jsx
+    const RoleArray = [
+      'developer',
+      'designer',
+      'researcher',
+      'project manager',
+      'other',
+    ];
+
+    const roleTrigger = (
+      <Button buttonType="primary" buttonSize="standard">
+        Role
+      </Button>
+    );
+    const roleHeader = (
+      <h3 className="w-full font-nunito-sans text-lg uppercase tracking-wider">
+        Select your role
+      </h3>
+    );
+
+    const roleContent = (
+      <div className="flex max-w-md flex-wrap items-center justify-center gap-6 bg-grayscale-100 py-6 text-lg">
+        {RoleArray.map((role, i) => {
+          return (
+            <button
+              key={i}
+              className={`rounded-3xl border-2 border-solid border-black p-4 font-nunito-sans capitalize ${
+                i === 0 ? 'bg-black text-white' : 'bg-white text-black'
+              } `}
+            >
+              {' '}
+              {role}
+            </button>
+          );
+        })}
+      </div>
+    );
+
+    const roleFooter = (
+      <div className="flex h-10 w-full items-center justify-between">
+        <h3 className="font-nunito-sans text-lg">Step 1 out of 5</h3>
+        <div className="flex gap-4">
+          <Button buttonType="alternative" buttonSize="standard">
+            Previous
+          </Button>
+          <Button buttonType="primary" buttonSize="standard">
+            Next
+          </Button>
+        </div>
+      </div>
+    );
+    // Return all modals
+    return (
+      <div className="flex gap-4">
+        <Modal
+          trigger={EmptyTriggerBtn}
+          header={EmptyHeader}
+          content={EmptyContent}
+          footer={EmptyFooter}
+        />
+        <Modal
+          trigger={roleTrigger}
+          header={roleHeader}
+          content={roleContent}
+          footer={roleFooter}
+        />
+        <Modal
+          trigger={SubscribeTriggerBtn}
+          header={SubscribeHeader}
+          content={SubscribeContent}
+          footer={SubscribeFooter}
+        />
+        <Modal
+          trigger={ToSTriggerBtn}
+          header={ToSHeader}
+          content={ToSContent}
+          footer={ToSFooter}
+        />
+      </div>
+    );
+  },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?node-id=8769%3A6382',
+    },
+  },
+};
+
+export const EmptyModal: Story = {
```

Comment:
This should be named Default (same as Figma)
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/27841994/804f8fc1-5f65-404c-8e05-a07a07eb0d11)